script/get-release-level.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
After [PR#701](https://github.com/test-network-function/cnf-certification-test/pull/701) I think we can get rid of the `oc` command installed in the `Dockerfile`, as well as the `OPENSHIFT_VERSION` variable and therefore this new script wouldn't be needed.

Reply:
Excellent!  I'll remove it.